src/lib/esnext.weakref.d.ts

```diff
@@ -0,0 +1,55 @@
+interface WeakRef<T extends object> {
+    readonly [Symbol.toStringTag]: "WeakRef";
+
+    /**
+     * Returns the WeakRef instance's target object, or undefined if the target object has been
+     * reclaimed.
+     */
+    deref(): T | undefined;
+}
+
+interface WeakRefConstructor {

```

Comment:
Hi, any chance to expedite this change?

Reply:
Yes, it got lost in my email, but I think it's good to go after I re-run the github CI.